contrib/mixin/mixin.libsonnet

```diff
@@ -238,6 +238,20 @@
               summary: 'etcd cluster database growing very fast.',
             },
           },
+          {
+            alert: 'etcdBackendHighFragmentationRatio',
+            expr: |||
+              (etcd_mvcc_db_total_size_in_use_in_bytes/etcd_mvcc_db_total_size_in_bytes)*100 < 50
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              description: 'etcd cluster "{{ $labels.%s }}": database size in use on instance {{ $labels.instance }} is less than half of the actual allocated disk space, please run defragmentation procedure to retrieve the unused fragmented disk space.' % $._config.clusterLabel,
```

Comment:
are there docs we can link or a runbook or something via `runbook_url` to help folks like me who will think "I have no idea how to run a defrag"?  Maybe [here][1], with some way to get the current version?  Or following [this pattern][2] so mixin consumers can slot in their preferred runbook source?  No worries if this is something that should get punted to follow-up work.

[1]: https://etcd.io/docs/v3.5/op-guide/maintenance/#defragmentation
[2]: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/a54540b92fb7b2d0826c0eff741e0894943babc7/lib/add-runbook-links.libsonnet#L13-L22

Reply:
I think we can use link below, cc @spzala 
https://etcd.io/docs/latest/op-guide/maintenance/ 

+1 to add `runbookURLPattern` pointed out above. Please feel free to address it in a followed PR.